cluster-up/cluster/k8s-provider-common.sh

```diff
@@ -14,7 +14,16 @@ function deploy_kwok() {
     fi
 }
 
-
+# copy_istio_cni_conf_files copy the generated Istio CNI net conf file
+# (at '/etc/cni/multus/net.d/') to where Multus expect CNI net conf files ('/etc/cni/net.d/')
+function copy_istio_cni_conf_files() {
+    if [ "$KUBEVIRT_DEPLOY_ISTIO" == "true" ] && [ "$KUBEVIRT_WITH_CNAO" == "true" ]; then
+        for nodeNum in $(seq -f "%02g" 1 $KUBEVIRT_NUM_NODES); do
+            $ssh node${nodeNum} -- "until ls /etc/cni/multus > /dev/null 2>&1; do sleep 1; done"
```

Comment:
Is there a danger here that it could sleep forever?

Reply:
its unlikely, since if istio is deployed with cnao the cni file then it will create this cni file. if it couldn't then it would probably crash before getting here

but i will add a timeout for extra safety

Reply:
Please do this change in separate PR.